requirements.txt

```diff
@@ -12,7 +12,7 @@ sqlalchemy==1.3.20
 sqlalchemy_utils==0.36.8
 camelot-py[cv]
 us==2.0.2
-TableauScraper==0.1.2
+TableauScraper==0.1.29
```

Comment:
I'm not sure what changes were made, but the NH and SC scrapers need this update to succeed

Reply:
Yeah. I noticed this when I was poking at the NC one. 🤷